explorer/rest/tests/test_rest.py

```diff
@@ -0,0 +1,161 @@
+			print(f'Creating database config ini file {db_config_path}...')
+			db_config_file.write('[nem_db]\n')
+			db_config_file.write('database = test\n')
+			db_config_file.write('user = postgres\n')
+			db_config_file.write('password = \n')
+			db_config_file.write('host = 127.0.0.1\n')
+			db_config_file.write('port = 5433\n')
+
+		with open(file_name, 'wt', encoding='utf8') as config_file:
+			print(f'creating config file {file_name}...')
+			config_file.write(f'DATABASE_CONFIG_FILEPATH="{db_config_path}"\n')
+			config_file.flush()
+
+			temp_file_path = file_name.resolve()
+			os.environ['EXPLORER_REST_SETTINGS'] = str(temp_file_path)
+			return create_app()
+
+
+@pytest.fixture
+def client(app):  # pylint: disable=redefined-outer-name
+	return app.test_client()
+
+# endregion
+
+
+# region /block/<height>
+
+
+def _assert_get_api_nem_block_by_height(client, height, expected_status_code, expected_result):  # pylint: disable=redefined-outer-name
+	# Act:
+	response = client.get(f'/api/nem/block/{height}')
+
+	# Assert:
+	assert expected_status_code == response.status_code
+	assert expected_result == response.json
+
+
+def test_api_nem_block_by_height(client):  # pylint: disable=redefined-outer-name
+	_assert_get_api_nem_block_by_height(client, 1, 200, BlockView(*BLOCKS[0]).to_dict())
+
+
+def test_api_nem_block_non_exist(client):  # pylint: disable=redefined-outer-name
+	_assert_get_api_nem_block_by_height(client, 3, 404, {
+		'message': 'Resource not found',
+		'status': 404
+	})
+
+
+def test_api_nem_block_by_invalid_height(client):  # pylint: disable=redefined-outer-name
+	_assert_get_api_nem_block_by_height(client, 'invalid', 400, {
+		'message': 'Bad request',
+		'status': 400
+	})
+
+# endregion
+
+
+# region /blocks
+
+def _get_api_nem_blocks(client, **query_params):  # pylint: disable=redefined-outer-name
+	query_string = '&'.join(f'{key}={val}' for key, val in query_params.items())
+	return client.get(f'/api/nem/blocks?{query_string}')
+
+
+def _assert_get_api_nem_blocks(client, expected_code, expected_result, **query_params):  # pylint: disable=redefined-outer-name
+	# Act:
+	response = _get_api_nem_blocks(client, **query_params)
+
+	# Assert:
+	assert expected_code == response.status_code
+	assert expected_result == response.json
+
+
+def _assert_get_api_nem_blocks_fail(client, **query_params):  # pylint: disable=redefined-outer-name
+	# Act:
+	response = _get_api_nem_blocks(client, **query_params)
+
+	# Assert:
+	assert 400 == response.status_code
+	assert {
+		'message': 'Bad request',
+		'status': 400
+	} == response.json
+
+
+def test_api_nem_blocks_without_params(client):  # pylint: disable=redefined-outer-name
+	# Act:
+	response = client.get('/api/nem/blocks')
+
+	# Assert:
+	assert 200 == response.status_code
+	assert [BlockView(*block).to_dict() for block in BLOCKS] == response.json
+
+
+def test_api_nem_blocks_limit_to_1(client):  # pylint: disable=redefined-outer-name
```

Comment:
probably would rename to `limit_to_1` to `applies_limit` (and same for similar tests)